### PR TITLE
RSS feeds for planet

### LINKED ISSUE #1: Add RSS feed
I'd like to be able to make some fancy IFTTT scripts, like popping these articles into Pocket automatically. Needs an RSS feed to do that kind of stuff.

### PR DESCRIPTION
**Pull request by cmcavoy**
Fixes toolness/planet-badges#1. Ready for review.

### CODE REVIEW

**Comment by cmcavoy** [comment]
hey @toolness this is ready for review whenever.

**Comment by toolness** [comment]
Oh snap, I'm looking at this now.

**Comment by toolness** [comment]
This looks good, though do you think we can make the origin configurable through, say, an `ORIGIN` environment variable rather than being hard-coded to http://planet.openbadges.org/? And also document that environment variable in the README?

**Comment by cmcavoy** [comment]
got it...updated with an origin variable!

File: build.js [hunk 1]
```diff
@@ -31,7 +34,34 @@ function parseFeed(name, url, cb) {
...
 }
 
+function renderRSS(context) {
+  // create the feed
+  var feed = new RSS();
+  feed.title = 'Planet Badges';
+  feed.description = 'A feed aggregator for Open Badges blogs';
+  feed.feed_url = 'http://planet.openbadges.org/rss';
+  feed.site_url = 'http://planet.openbadges.org';
+  feed.image_url = 'http://planet.openbadges.org/badge-logo.png';
```
Comment (toolness) [inline build.js]: Shouldn't these three lines incorporate the `ORIGIN` environment variable?

**Comment by toolness** [comment]
Woot thanks!

**Pull request merged**